streams/src/main/java/org/apache/kafka/streams/state/internals/StoreQueryUtils.java

```diff
@@ -333,6 +339,43 @@ public static boolean isPermitted(
         return byteArray -> deserializer.deserialize(serdes.topic(), byteArray);
     }
 
+
+    public static Map<TopicPartition, Long> positionToTopicPartitionMap(final Position position) {
```

Comment:
This can be private, right?

Reply:
Why don't we add serialize and deserialize methods to the `Position`?

Reply:
Or, move these methods to the `Position` class

Reply:
@vpapavas my understanding is we don't want to add extra serdes to public "API" classes.